src/components/KModalFullscreen/KModalFullscreen.vue

```diff
@@ -302,42 +303,43 @@ $fullscreen-modal-padding: 64px;
 .k-modal-fullscreen-title {
   display: inline-flex;
   justify-content: center;
-  margin-bottom: var(--spacing-xs, spacing(xs));
+  margin-bottom: var(--spacing-xs, var(--kui-space-40, $kui-space-40));
   position: relative;
 
-  @media (min-width: $viewport-md) {
+  @media (min-width: $kui-breakpoint-phablet) {
     justify-content: flex-start;
-    margin-bottom: 0;
-    margin-left: 36px;
+    margin-bottom: var(--kui-space-0, $kui-space-0);
+    margin-left: var(--kui-space-90, $kui-space-90); // This was 36px, switched it to 32px
   }
 }
 
 .k-modal-fullscreen-action {
   display: inline-flex;
   justify-content: center;
-  margin-right: var(--spacing-xl, spacing(xl));
+  margin-left: var(--kui-space-50, $kui-space-50) !important;
+  margin-right: var(--spacing-xl, var(--kui-space-90, $kui-space-90));
 
   & button,
   & :deep(button) {
-    font-size: 13px;
-    font-weight: 600;
-    height: 40px;
-    line-height: 13px;
-    margin-left: var(--spacing-md, spacing(md));
+    font-size: var(--kui-font-size-20, $kui-font-size-20);
+    font-weight: var(--kui-font-weight-semibold, $kui-font-weight-semibold);
+    height: var(--kui-space-100, $kui-space-100);
+    line-height: var(--kui-line-height-10, $kui-line-height-10);
+    margin-left: var(--spacing-md, var(--kui-space-60, $kui-space-60));
   }
 
-  @media (min-width: $viewport-md) {
+  @media (min-width: $kui-breakpoint-phablet) {
     justify-content: flex-end;
   }
 }
 
 .k-modal-fullscreen-body-header,
 .k-modal-fullscreen-body {
-  color: var(--KModalFullscreenColor, var(--black-500, color(black-500)));
-  padding-left: var(--spacing-lg);
-  padding-right: var(--spacing-lg);
+  color: var(--KModalFullscreenColor, var(--black-500, var(--kui-color-text, $kui-color-text)));
+  padding-left: var(--spacing-lg, var(--kui-space-80, $kui-space-80));
+  padding-right: var(--spacing-lg, var(--kui-space-80, $kui-space-80));
 
-  @media (min-width: $viewport-md) {
+  @media (min-width: $kui-breakpoint-phablet) {
     padding-left: 120px;
     padding-right: 120px;
```

Comment:
```suggestion
    padding-left: $tmp-spacing-120;
    padding-right: $tmp-spacing-120;
```

Reply:
Same for other `$tmp` variables. But if we decide it doesn't become a token, we can replace it's usages easily later.

Reply:
This file also have:
```
 padding-left: 230px;
 padding-right: 230px;
```

Reply:
I would create and use `$tmp` variables for those as well. Like I said above, it's easy to drop them later and switch back to pixel values.

Reply:
I don't think it makes sense to map these one-off sizes to temporary variables. Are they used anywhere else in the repo? If no, they shouldn't need a variable.

Reply:
I don't see 230px anywhere else, 120px is being used in 2 components..

Reply:
Maybe that's true for `230px`, but 120px is used somewhere else (I don't remember exactly where, but I created and used it). The only way to know that it's a "one off" value is if the $tmp variable created for that value is only used once. When I or Aanchal encounters `230px` for the next time, we are not gonna remember that we had this value somewhere els=se before therefore now it deserves it's own `$tmp` variable.

Reply:
I'm not sure I understand the question... they both use value `120px` but they are not related as they live in different components
> But are they used in two places because the values are directly related, or because they just happen to use the same value?

Reply:
If the temporary token that points to 120px changes to 220px, should both components change?

Reply:
Just looked up, `$tmp-spacing-120` is used for positioning in one other component. Safe to say it's not needed there (I will fix that one occurrence in my next PR). Since there are no other usages of `120px`, I'm okay with leaving this a pixel value (same for `230px`).